scripts/scripts-1.js

```diff
@@ -0,0 +1,171 @@
+function getMinNumberFromParams(numb_1, numb_2, numb_3, numb_4) {
+
+  const minimalNumber = Math.min(numb_1, numb_2, numb_3, numb_4);
+
+  if (numb_1 === undefined || numb_2 === undefined || numb_3 === undefined || numb_4 === undefined) {
+
+    return 'Undefined value!';
+
+  } else if (numb_1 === '' || numb_2 === '' || numb_3 === '' || numb_4 === '') {
+
+    return 'Your value is empty';
+
+  } else if (isNaN(minimalNumber)) {
```

Comment:
And you should validate params, not the result.
It's not okay that the function works with other data types
<img width="566" alt="image" src="https://user-images.githubusercontent.com/84010501/191553634-9f8095e9-f52c-4b26-b786-fb18b6f2f64c.png">